src/main/java/org/jclouds/vcloud/director/v1_5/domain/CapacityWithUsage.java

```diff
@@ -25,25 +25,24 @@
 import com.google.common.base.Objects;
 
 /**
- * Represents a capacity and usage of a given resource.
- * <p/>
- * <p/>
+ * <p>Represents a capacity and usage of a given resource.
+ * <p>&nbsp;</p>
  * <p>Java class for CapacityWithUsage complex type.
- * <p/>
+ * <p>&nbsp;</p>
  * <p>The following schema fragment specifies the expected content contained within this class.
- * <p/>
+ * <p>&nbsp;</p>
  * <pre>
- * &lt;complexType name="CapacityWithUsage">
- *   &lt;complexContent>
- *     &lt;extension base="{http://www.vmware.com/vcloud/v1.5}CapacityType">
- *       &lt;sequence>
- *         &lt;element name="Used" type="{http://www.w3.org/2001/XMLSchema}Long" minOccurs="0"/>
- *         &lt;element name="Overhead" type="{http://www.w3.org/2001/XMLSchema}Long" minOccurs="0"/>
- *       &lt;/sequence>
- *       &lt;anyAttribute processContents='lax' namespace='##other'/>
- *     &lt;/extension>
- *   &lt;/complexContent>
- * &lt;/complexType>
+ * &lt;complexType name="CapacityWithUsage"&gt;
```

Comment:
@bostko Do we really need to use `&lt;` and `&gt;`? Can we not do the same as in https://github.com/apache/brooklyn-server/blob/master/core/src/main/java/org/apache/brooklyn/feed/ssh/SshFeed.java#L62-L63 (with the `<pre>` and `{@code ... }`? That seems to work for containing `<` and `>` in the javadoc.